Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI.js

```diff
@@ -79,6 +63,14 @@ getAdvancedAuthMethodHeaders = function(key, auth_id, content_type,) {
 
 getRequestURL = function (uri) {
     var requestUrl = serverURL;
+
+    // only when using XSIAM or XSOAR >= 8.0 we will add the /xsoar suffix
+    // and only when it is not a system endpoint (note: this list does not include all system endpoints!)
```

Comment:
>XSOAR >= 8.0

Can we say for sure that in XSOAR 8 we need that prefix as well? have we checked it?
>this list does not include all system endpoints!)

WDYM by this? What list?